po/es.po

```diff
@@ -1559,29 +1552,29 @@ msgstr "%sCargando %s..."
 #: src/reloader.cpp:105 src/reloader.cpp:110 src/reloader.cpp:115
 #, c-format
 msgid "Error while retrieving %s: %s"
-msgstr "Error al obtener %s: %s"
+msgstr "Error al descargar %s: %s."
 
 #: src/reloader.cpp:125
 msgid "Error: invalid feed!"
-msgstr "Error: ¡fuente inválida!"
+msgstr "Error: fuente incorrecta."
```

Comment:
Ditto about punctuation.

Reply:
Withdrawn, [see explanation in a comment below](https://github.com/newsboat/newsboat/pull/874#issuecomment-612408710).